Makefile.am

```diff
@@ -323,9 +297,10 @@ endif # SERVER
 
 endif # SIEVE
 
-EXTRA_DIST += \
+EXTRA_DIST = \
```

Comment:
Leave this a `+=` with the initialisation back at the top, then if we need to add back more conditional stuff we don't have to chase the declaration around.